### PR TITLE
Qa 0.19

### PR DESCRIPTION
**Pull request by crssnd**
**What this PR does / why we need it**: QA changes and updates

### CODE REVIEW

**Review by OlleLarsson — APPROVED** [review]

File: helmfile/values/kube-prometheus-stack-sc.yaml.gotmpl [hunk 1]
```diff
@@ -25,7 +25,8 @@ kubeApiServer:
...
       - action: keep
         sourceLabels: [__name__]
-        regex: '(process_cpu_seconds_total|process_resident_memory_bytes|go_goroutines|workqueue_queue_duration_seconds_bucket|workqueue_adds_total|workqueue_depth|apiserver_request_total|apiserver_request_count|apiserver_request_latencies_bucket|apiserver_client_certificate_expiration_seconds_count|apiserver_request_duration_seconds_sum|apiserver_request_duration_seconds_count|apiserver_request_duration_seconds_bucket|APIServiceRegistrationController_depth)'
+        regex: '(process_cpu_seconds_total|process_resident_memory_bytes|go_goroutines|workqueue_queue_duration_seconds_bucket|workqueue_adds_total|workqueue_depth|apiserver_request_total|apiserver_request_count|apiserver_request_latencies_bucket|apiserver_client_certificate_expiration_seconds_count|apiserver_request_duration_seconds_sum|apiserver_request_duration_seconds_count|APIServiceRegistrationController_depth)'
+#        regex: '(apiserver_request_count|apiserver_request_latencies_bucket|apiserver_client_certificate_expiration_seconds_count|apiserver_request_duration_seconds_sum|apiserver_request_duration_seconds_count|APIServiceRegistrationController_depth)'
```
Comment (viktor-f) [inline helmfile/values/kube-prometheus-stack-sc.yaml.gotmpl:29]: Why is there a commented line here?

File: helmfile/values/prometheus-alerts-sc.yaml.gotmpl [hunk 2]
```diff
@@ -6,6 +6,12 @@ defaultRules:
   rules:
     alertmanager: true
     rookMonitor: {{ .Values.monitoring.rook.enabled }}
+    {{- if and .Values.thanos.enabled .Values.thanos.metrics.enabled }}
     thanosMonitor: {{ .Values.thanos.metrics.enabled }}
+    {{- end }}
+    {{- if not .Values.prometheus.predictLinearLimit }}
+    predictLinear: false
+    {{- end }}
```
Comment (viktor-f) [inline helmfile/values/prometheus-alerts-sc.yaml.gotmpl:14]: Will this be set to false if you set `predictLinearLimit: 0` or is it `predictLinearLimit: null`? Either way, could you add a comment in wc-config and sc-config to explain this option there?

Comment (aarnq) [reply helmfile/values/prometheus-alerts-sc.yaml.gotmpl:14]: I've updated the comment about that option.

File: helmfile/values/kube-prometheus-stack-sc.yaml.gotmpl (hunk 1 above)
Comment (aarnq) [reply helmfile/values/kube-prometheus-stack-sc.yaml.gotmpl]: This is something that I assume was left after testing, so I removed it.

**Review by viktor-f — APPROVED** [review]
LGTM

**Pull request merged**